web/Views/Admin/AdminList.cshtml

```diff
@@ -52,6 +53,12 @@
             <tr>
                 <td>@currentAdmin.GoogleEmail</td>
                 <td>@currentAdmin.GoogleUserName</td>
+                <td>
+                    @using (Html.BeginForm("RemoveAdmin", "Admin", new { userToRemoveId = currentAdmin.Id }, FormMethod.Post))
+                    {
+                        <button type="submit" class="btn btn-outline-danger">Remove</button>

```

Comment:
I'm wondering if it might be clearer to have this button say "Revoke" or similar seeing as we're not removing the user, just removing their admin status. What do you think?

Reply:
That's true - I'll reword it.